app/controllers/AdminUsers/employees_controller.rb

```diff
@@ -25,6 +25,30 @@ def destroy
       redirect_to admin_users_employees_path, notice: 'Employee was successfully destroyed.'
     end
 
+    def edit_number_of_available_kudos_for_all
```

Comment:
do we really need that action? we do not even need the employees parameter - all you need is a form that allows to select a number with a submit button passing that value to the next action :)

Reply:
No, we don't :D I have removed render action from this method ;)